diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 881e680..899b39f 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -79,13 +79,10 @@ add_core("bind_management")
 add_core("device_information")
 add_core("attribute_reporting")
 add_core("group_table")
+add_core("group_management")
 add_core("session_management")
 add_core("rssi")
 
-if(${HF_GROUP_SUPPORT})
-   add_core("group_management")
-endif()
-
 if(${HF_CORE_SUOTA_SUPPORT})
    add_core("suota")
 endif()
